engine/schema/src/main/java/com/cloud/network/dao/LoadBalancerVO.java

```diff
@@ -136,4 +136,9 @@ public Scheme getScheme() {
     public String getCidrList() {
         return cidrList;
     }
+
+    @Override
+    public String toString() {
+        return String.format("Load Balancer {uuid: \"%s\", name: \"%s\"}", getUuid(), name);
```

Comment:
We can use `ReflectionToStringBuilderUtils` here.

Reply:
@GutoVeronezi 
I think `ReflectionToStringBuilderUtils` will not work properly here, since `LoadBalancerVO` doesn't have the `uuid` attribute (it's inherited from `FirewallRuleVO`). So I've used `String.format()` getting the `uuid` from the getter method.

Reply:
`ReflectionToStringBuilderUtils` uses `ReflectionUtils.getAllFields`, which will also retrieve fields of the super types too; therefore it will work properly.

Reply:
@GutoVeronezi done!